crates/router/src/connector/braintree/transformers.rs

```diff
@@ -1203,12 +1203,6 @@ pub struct CaptureInputData {
     transaction: CaptureTransactionBody,
 }
 
-#[derive(Debug, Clone, Serialize)]
```

Comment:
Dead Code

Reply:
Remove this change. Only relevant changes for adding WellsFargoPayout template should be included in this PR.